st2client/st2client/commands/action.py

```diff
@@ -1343,19 +1343,28 @@ def __init__(self, resource, *args, **kwargs):
 
     @add_auth_token_to_kwargs_from_cli
     def run(self, args, **kwargs):
-        return self.manager.pause(args.id)
+        responses = []
+        for execution_id in args.id:
+            response = self.manager.pause(execution_id)
+            responses.append([execution_id, response])
+
+        return responses
 
     @add_auth_token_to_kwargs_from_cli
     def run_and_print(self, args, **kwargs):
-        try:
```

Comment:
Where's the try block now?

Reply:
I've moved this around. I think it makes the most sense in the `run()` function because this is where the call to retrieve the resource is made. Putting the call in the `for` loop allows us to capture the error for the individual execution and re-raise it with a message containing the execution ID that failed.